src/nectarchain/dqm/start_calib.py

```diff
@@ -16,11 +16,27 @@
 from trigger_statistics import TriggerStatistics
 from camera_monitoring import CameraMonitoring
 
+import argparse
+
+# Create an ArgumentParser object
+parser = argparse.ArgumentParser(description='My script')
+parser.add_argument('-plot', action='store_true', help='Whether to plot the data')
+parser.add_argument('input_paths', nargs='+', help='Input paths')
+args = parser.parse_args()
+
+#Reading arguments, paths and plot-boolean
+path = args.input_paths[0]
+print("Input file paths:")
+print(path)
+for arg in args.input_paths[1:]:
+    print(arg)
+PlotFig = args.plot
+print("Plot:", PlotFig)
+
+#NectarPath
+NectarPath = str(os.environ['NECTARDIR'])
```

Comment:
nectardir is pointing towards the folder that contains the nectarchain software on my local machine. It can be defined by each user as they whISH. Here it is used to create the putput folder in the same directory where all my nectarchain work is contained. I have no strong preferences here, so as you see fit

Reply:
My point is to try avoiding relying on hidden environment variables as much as possible. If related to the output directory for results and plot, it would be better to pass them through arguments in `start_calib.py`, in my opinion.

Reply:
done, now start_clib takes input, output, files and (plotting) options in argument. 
I will precise that in the readme